.travis.yml

```diff
@@ -62,6 +63,7 @@ before_script:
   # Prepare system
   - if [ "$TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_unittest.sh ; fi
   - if [ "$BEHAT_OPTS" != "" ] || [ "$REST_TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_behat.sh ; fi
+  - cd "$HOME/build/ezplatform"; docker-compose exec --user www-data app sh -c "rm -rf var/cache/behat/*"
```

Comment:
This looks like a hack, is cache clearing or SYMFONY_ENV not set correctly in meta repo maybe?  /cc @alongosz